execution.go

```diff
@@ -291,14 +294,16 @@ func (q *queryExecution) createGQLErrors(step *QueryPlanStep, err error) gqlerro
 // have insertionPoint: ["foo", "bar", "movies", "movie", "compTitles"], with the below example as the boundary result we're
 // crawling for ids:
 // [
-// 	 {
```

Comment:
can you remove the indentation/whitespace changes?

Reply:
I'm getting the changes just from running `gofmt`. Do you mind if they stay?

Reply:
`gofmt` really shouldn't be touching comments, but can you pull them out into a different commit then?